test/test-actions.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+var child_process = require('child_process')
+var test = require('tape')
+var path = require('path')
+
+var execFileSync = child_process.execFileSync
+var execFile = child_process.execFile
+
+var addonPath = path.resolve(__dirname, 'node_modules', 'name\'s (weird)')
+var nodeGyp = path.resolve(__dirname, '..', 'bin', 'node-gyp.js')
+
+function runSpacedName() {
+  var testCode = "require('name\\'s (weird)')"
```

Comment:
You can't put a `\` in a Windows path. But you can put an unescaped `'`.
You could try using `require("${addonPath}")` but YMMV